scripts/js/upgrade/index.js

```diff
@@ -0,0 +1,330 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+const { blake2AsHex } = require('@polkadot/util-crypto');
+const fs = require('fs')
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+const run = async () => {
+  let exitCode = 0;
+  try {
+    console.log("Parsing Args ...")
+    // 0 & 1 are command context
+    const endpoint = "ws://0.0.0.0:9946";
+    const endpointRelay = "ws://0.0.0.0:9944";
+    const seeds = ["//Alice", "//Bob", "//Charlie"];
```

Comment:
`seeds` is just used to access alice, bob, and charlie, so I would just make those items const themselves and reference them directly.

```suggestion
    const ALICE = "//Alice";
    const BOB =  "//Bob";
    const CHARLIE = "//Charlie";
```